frontend/app/src/components/display/slide-over.tsx

```diff
@@ -1,11 +1,12 @@
 import { Dialog, Transition } from "@headlessui/react";
-import React, { Fragment, useRef } from "react";
+import React, { Fragment, useRef, useState } from "react";
 import { Badge } from "@/components/ui/badge";
 import { Icon } from "@iconify-icon/react";
 import { ObjectHelpButton } from "@/components/menu/object-help-button";
 import { useAtomValue } from "jotai/index";
 import { currentBranchAtom } from "@/state/atoms/branches.atom";
 import { IModelSchema } from "@/state/atoms/schema.atom";
+import ModalConfirm from "../modals/modal-confirm";
```

Comment:
Why ModalConfirm instead of ModalDelete? The green button made me think twice. I feel like a red button is more explicit that's it destructive operation. WDYT?